rbc/omniscidb.py

```diff
@@ -621,6 +623,8 @@ def is_udtf(sig):
                             annot = a.annotation()
                             _sizer = annot.get('sizer', unspecified)
                             if _sizer is not unspecified:
+                                # sizer must have type int32
+                                assert a.is_int and a.bits == 32, a
```

Comment:
So, maybe add an error message `sizer must have type int32` in case the assertion fails.